src/style.css

```
@@ -0,0 +1,32 @@
+body {
+  margin: 0;
+  background-color: black;
+}
+
+.blue {
+  width: 348px;
+  margin: 20px 20px 40px;
+  border: solid;
+  border-width: 1px;
+  border-color: #e75551;
```

```suggestion
  border: 1px solid #e75551;
```
use short property version